src/lib/Node.ts

```diff
@@ -0,0 +1,211 @@
+import { EventEmitter } from 'events';
+import NodeSocket from './Structures/NodeSocket';
+import NodeServer from './Structures/NodeServer';
+import { NodeOptions, SendOptions, BroadcastOptions } from 'veza';
+
+class Node extends EventEmitter {
+
+	/**
+   * @typedef {Object} NodeOptions
```

Comment:
This comment and many others are pretty broken, *maybe someday ESLint will lint JSDoc's indentations*.
```suggestion
	 * @typedef {Object} NodeOptions
```

Reply:
Done. 👍

Reply:
Not quite yet, there are still lines mixing tabs and spaces. In GitHub it's pretty easy to find them as the tabs take 8 spaces, and if you look at the diff, you can find them easily.

My tricks are to select everything in the very middle, if there's a tab it'll select either the start of the line or the end of the padding, and if there's a space, it'll select the middle of it. Kind of like this:

![2019-05-20_19-40-05](https://user-images.githubusercontent.com/24852502/58040931-13603280-7b37-11e9-8bae-b30b030c7509.gif)

Reply:
Ok, I'll look into it. 
What do you want me use ? Tabs or spaces ?
Adding 2 tabs messed up the alignment, that's why I had to use tab+space.

Reply:
I prefer tabs. Also, there's a tool in Visual Studio Code to convert all spaces to tabs, though I'm not sure if it also converts the padding from comment blocks.

- `Ctrl + Shift + P`.
- `> Convert indentation to Tabs`.

Reply:
Is this fine ?

Reply:
There's some weird `\t\s\t` that should be `\t\t\s`, but that'd be nitpicking a lot. Do the tests still work after this rewrite? They're not run by the CI yet.

Reply:
No, the tests won't work since there were some issues I couldn't resolve properly.

Reply:
Oh, ok, I'll merge this later and fix them, again, thanks you very much for this!